orienteer-core/src/main/java/org/orienteer/core/web/FlatMenuPanel.java

```diff
@@ -0,0 +1,103 @@
+package org.orienteer.core.web;
+
+import java.util.List;
+
+import org.apache.wicket.AttributeModifier;
+import org.apache.wicket.behavior.AttributeAppender;
+import org.apache.wicket.markup.html.basic.Label;
+import org.apache.wicket.markup.html.link.ExternalLink;
+import org.apache.wicket.markup.html.list.ListItem;
+import org.apache.wicket.markup.html.list.ListView;
+import org.apache.wicket.markup.html.panel.GenericPanel;
+import org.apache.wicket.model.IModel;
+import org.apache.wicket.model.PropertyModel;
+import org.apache.wicket.request.cycle.RequestCycle;
+import org.orienteer.core.model.ODocumentNameModel;
+
+import com.orientechnologies.orient.core.record.impl.ODocument;
+
+import ru.ydn.wicket.wicketorientdb.model.ODocumentPropertyModel;
+
+/**
+ * 
+ * Flat menu panel with dropdown support
+ *
+ */
+public class FlatMenuPanel extends GenericPanel<ODocument>{
+	private static final long serialVersionUID = 1L;
+	private String itemsFieldName;
+
+	public FlatMenuPanel(String id, IModel<ODocument> itemModel, String itemsFieldName) {
+		super(id, itemModel);
+        this.itemsFieldName = itemsFieldName;
+        setOutputMarkupId(true);
+        add(new ListView<ODocument>("items", new PropertyModel<List<ODocument>>(this, "items")) {
+
+            @Override
+            protected void populateItem(ListItem<ODocument> item) {
+                IModel<ODocument> itemModel = item.getModel();
+                ODocumentPropertyModel<String> urlModel = new ODocumentPropertyModel<String>(itemModel, "url");
+                ODocumentPropertyModel<List<ODocument>> subItems = new ODocumentPropertyModel<List<ODocument>>(itemModel, "subItems");
+                final boolean hasSubItems = subItems.getObject() != null && !subItems.getObject().isEmpty();
+                ExternalLink link = new ExternalLink("link", urlModel)
+                        .setContextRelative(true);
+                link.add(new Label("name", new ODocumentNameModel(item.getModel())).setRenderBodyOnly(true));
+                item.add(link);
+                if (isActiveItem(urlModel)) {
+                   	link.add(new AttributeAppender("class", " active"));
+                }
+                if (hasSubItems){
+                   	link.add(new AttributeModifier("aria-expanded", "false"));
```

Comment:
Why these lines needed? It can be coded right in HTML.

Reply:
If i do - all links show as dropdowns